build/Dockerfile

```diff
@@ -76,24 +76,12 @@ ARG SERVER_PORT=10999
 ENV DESCRIPTION="Powered by DST Academy." \
 	SERVER_PORT="$SERVER_PORT" \
 	MAX_PLAYERS=4 \
-	WHITELIST_SLOTS=0 \
-	PVP_ENABLE=false \
 	GAME_MODE=survival \
```

Comment:
No, it's in the other way. `INTENTION` depends on `GAME_MODE`. So `GAME_MODE` is the important one.